demos/instr.cc

```diff
@@ -57,9 +57,7 @@ static void LFence() {
     defined(_M_IX86)
   _mm_lfence();
 #elif defined(__aarch64__)
-  asm volatile(
-      "dsb ld\n"
-      "isb\n");
+  asm volatile("dsb ld");
```

Comment:
I think the difficulty we're having here is because we're trying to graft one architecture's abstractions (`mfence` and `lfence` in x86) onto other platforms.

Echoing what I said in #36, I expect the right answer is instead to write one version of `ReadLatency` for each architecture. Further evidence: our current version doesn't insert an `isb` *before* the call to `rdtsc`, which means we are probably getting a stale value of `cntvct_el0` and our timing is off.

Happily, if we write per-architecture `ReadLatency` we only need to use one `asm` syntax (for now) since we only support MSVC on x86 and there we have Intel's intrinsics.